samples/apps/react-vanilla-sample/src/theme/theme.css

```
@@ -17,39 +17,48 @@
+    --md-contained-primary-hover-background: linear-gradient(90deg, #2572d3 0%, #009cb8 100%);
 }
 
 html[data-color-scheme='light'] {
     color-scheme: light;
-
-    --md-palette-background-paper: #ffffff6b;
-    --md-palette-border-color-paper: #ffffffd6;
-
+    --md-palette-background-paper: rgba(244, 246, 249, 0.7);
+    --md-palette-border-color-paper: rgba(191, 198, 207, 0.4);
     background: var(--background);
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)